components/list/list-item.js

```diff
@@ -2,6 +2,8 @@ import 'fastdom/fastdom.js';
 import { css, html, LitElement } from 'lit-element/lit-element.js';
 import { checkboxStyles } from '../inputs/checkbox-shared-styles.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { LinkMixin } from '../link/link-mixin.js';
```

Comment:
Can we leave whether we want a mixin for links until I convert links next week? I'm not sure we actually want one, since if we have shared styles people can apply link styles to any `<a>` element they want. And for places like this that want to expose some link properties, I think exposing them all isn't typically what they want. List items can have a `href`, but a lot of the other properties don't make sense.

Reply:
Sounds good to me.